### ISSUE TITLE
Question regarding validation and test set

### ISSUE THREAD

**Issue by NiranjanRavi1993** [Eaphan/BiProDet#18]
Hi,
    Thank you for the excellent repo. I used multi gpus to train the model from scratch and when I check the performance of the network on validation set, it is little lower compared to the pre-trained checkpoint provided. 
    Any additional changes/suggestions for network training would be really helpful. 
Regards
Niranjan.
